extensions/VisualEditor/wikia/modules/ve/ui/widgets/ve.ui.WikiaMediaOptionWidget.js

```diff
@@ -34,17 +34,17 @@ ve.ui.WikiaMediaOptionWidget = function VeUiWikiaMediaOptionWidget( config ) {
 	this.$previewIcon = this.$( '<span>' );
 	this.$previewText = this.$( '<span>' );
 	// TODO: Presence of checkbox perhaps should depend on the configuration
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)